src/Infrastructure/Persistence/Migrations/ApplicationDbContextModelSnapshot.cs

```diff
@@ -238,6 +238,307 @@ protected override void BuildModel(ModelBuilder modelBuilder)
 
                     b.ToTable("trust_levels");
                 });
+
+            modelBuilder.Entity("Codidact.Infrastructure.Identity.ApplicationUser", b =>
+                {
+                    b.Property<string>("Id")
+                        .HasColumnName("id")
+                        .HasColumnType("text");
+
+                    b.Property<int>("AccessFailedCount")
+                        .HasColumnName("access_failed_count")
+                        .HasColumnType("integer");
+
+                    b.Property<string>("ConcurrencyStamp")
+                        .IsConcurrencyToken()
+                        .HasColumnName("concurrency_stamp")
+                        .HasColumnType("text");
+
+                    b.Property<string>("Email")
+                        .HasColumnName("email")
+                        .HasColumnType("character varying(256)")
+                        .HasMaxLength(256);
+
+                    b.Property<bool>("EmailConfirmed")
+                        .HasColumnName("email_confirmed")
+                        .HasColumnType("boolean");
+
+                    b.Property<bool>("LockoutEnabled")
+                        .HasColumnName("lockout_enabled")
+                        .HasColumnType("boolean");
+
+                    b.Property<DateTimeOffset?>("LockoutEnd")
+                        .HasColumnName("lockout_end")
+                        .HasColumnType("timestamp with time zone");
```

Comment:
Does this need `with time zone`? We should be storing datetimes as UTC in the database, regardless of application or client time zone.

Reply:
I checked https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.identity.identityuser-1.lockoutend?view=aspnetcore-3.1
It says that it saves it as UTC.

I am just not sure if this will create a bug or not. EF automatically maps DateTimeOffset to timestamp with time zone. I can configure it to no time zone but I fear it will have unforeseen consequences.
I tried building a test for this but then realized I can't do in memory testing since I need the behavior of the schema tested.